torch/onnx/utils.py

```diff
@@ -76,6 +76,10 @@ def export(model, args, f, export_params=True, verbose=False, training=None,
            do_constant_folding=True, example_outputs=None, strip_doc_string=True,
            dynamic_axes=None, keep_initializers_as_inputs=None, custom_opsets=None,
            enable_onnx_checker=True, use_external_data_format=False):
+    import builtins
```

Comment:
Can you move this part of patch code into the description?

Reply:
Sure, moved to description